### ISSUE TITLE
blows up if attributes is nil

### ISSUE THREAD

**Issue by treznick** [ContinuityControl/fdic#2]
When trying to get an institution with an invalid certificate number, the call to `resp['d']['results']` will return `nil`, which is not handled in `FDIC::Record`. We should return `nil`, or raise something or do something else cool if results is an empty array.

**Comment by treznick** [comment]
Some thoughts I have re: this problem, and the architecture already:

We have two models to represent Financial Institutions that the FDIC covers: `Bank` and `Institution`. Thinking about this RESTfully, a GET to `Bank` with query params will return different `Banks`, but these are fairly shallow objects (it's just a set of the query-able fields and some identifiers). Making a GET request to `Institution` returns an individual institution (which has all of the fields that a `Bank` has), but only if the certification number is correct (see above). Maybe we should decide to take a particular direction here: What if we structured the API to use `/Bank` like a GET request to query for multiple institutions (by name, city, state, or by certification number), and then for each `Bank` object, we just grabbed each `Institution` and returned them as the query results?

While this may make a few assumptions about the nature of the resources that the FDIC exposes (or may expose), it more closely mimics the BankFind UI, reduces some of the complexity exposed to end users (there's just one model to represent a financial institution) and avoids more complicated error handling to deal with bad queries (you just search for institutions through the `/Bank` endpoint, and if nothing is found, you return an empty set).

@danbernier @jnimety thoughts?
